features/mbedtls/platform/COMPONENT_PSA_SRV_IMPL/inc/default_random_seed.h

```diff
@@ -0,0 +1,45 @@
+
+
+#ifndef MBED_OS_RTOS_COMPONENT_PSA_SRV_IMPL_DEFAULTRANDOMSEED_H_
+#define MBED_OS_RTOS_COMPONENT_PSA_SRV_IMPL_DEFAULTRANDOMSEED_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef RANDOM_SEED_ITS_UID
+    #define MBED_RANDOM_SEED_ITS_UID 0xBAADA555
```

Comment:
This doesn't look random. Non-random values have a higher chance of collisions...